on-prem-docs/getting-started.md

```diff
@@ -25,7 +25,6 @@ The following are minimum requirements for installation/deployment of the Chef H
 * We recommend:
   * 50 GB disk space for trial deployments
   * 100 GB disk space for production deployments
-* Deploy services single-node - scale out is not yet supported
```

Comment:
Doesn't the newly supported configuration have some networking requirements?  Perhaps mention that we can run single-node or multi-node provided we have TCP connectivity between peers.

Reply:
Yes those details are in `scaling.md`. We do mention port requirements there. Orgs would need the port details in order to setup their firewalls or AWS security groups unless they were leaving them open which they often don't.